lua/packer/util.lua

```diff
@@ -94,4 +117,17 @@ util.deep_extend = function(policy, ...)
   return result
 end
 
+util.system_output = function(command)
```

Comment:
Is this intended for running system commands that should not be run asynchronously?

Reply:
Yeah, I want to wait on the output of `luarocks list lpeg` and what not to check if somethign is already installed.